CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to shiplift
+Contributing to shiplift isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help make shiplift better.
+
+## Working on shiplift
+### Prerequisites
+* [Git](https://git-scm.com/)
+
+### Getting shiplift
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/shiplift.git
+$ cd shiplift
+```
+
+### Please pay attention to
+1. open an issue describing the feature/bug you wish to contribute first to start a discussion, explain why, what and how
+2. follow the eslint rules
```

Comment:
This repository does currently use JavaScript. `rustfmt` would be more appropriate.  I'm thinking it would be good to take some inspiration from [wasm-pack here](https://github.com/rustwasm/wasm-pack/blob/master/CONTRIBUTING.md#configuring-rustfmt)

Reply:
added rustfmt guideline. is it visible that i'm mainly node/js developer? :) 
just started to learn rust though.